llvm/lib/Target/PowerPC/PPCAsmPrinter.cpp

```diff
@@ -830,7 +816,9 @@ void PPCAsmPrinter::emitInstruction(const MachineInstr *MI) {
     // For TLS initial-exec and local-exec accesses on AIX, we have one TOC
     // entry for the symbol (with the variable offset), which is differentiated
     // by MO_TPREL_FLAG.
-    if (MO.getTargetFlags() & PPCII::MO_TPREL_FLAG) {
+    if (MO.getTargetFlags() == PPCII::MO_TPREL_FLAG ||
```

Comment:
It may be good to pull out `MO.getTargetFlags()` since we access it multiple times here.

Reply:
Done